source/doc-pages/introduction.md

```diff
@@ -1,35 +1,62 @@
-Ruby Object Mapper (ROM) is a Ruby library with the goal to provide powerful object
-mapping capabilities without limiting the full power of your datastore. ROM is based
-on several concepts and decisions that differentiate it from your normal Ruby ORM:
+Ruby Object Mapper (ROM) is a Ruby persistence library with the goal to provide 
```

Comment:
I don’t really feel strongly about either one, but I agree that `database` is less confusing than `datastore`—probably because it’s more concrete rather than abstract. `datastore` feels more correct conceptually.

The most important concern is to consistently use the same terms to refer to the same concepts throughout the high level docs.